examples/static_map/device_view_example.cu

```diff
@@ -28,7 +28,7 @@
 #include <limits>
 
 /**
- * @file device_side_example.cu
+ * @file device_view_example.cu
```

Comment:
Can we make the CI/`pre-commit` doxygen check aware for this kind of thing?

Reply:
Spent some time looking into this and find nothing useful.